rebasehelper/exceptions.py

```diff
@@ -26,7 +26,17 @@ class RebaseHelperError(Exception):
     Class representing Error raised inside rebase-helper after intentionally
     catching some expected and well known exception/error.
     """
-    pass
+    def __init__(self, msg, logfile=None):
+        """
+        Store rebase error into variables
+        :param msg: Rebase Helper error
+        :param logfile: Path to a logfile containing origin of the problem
+        """
+        if len(msg[0]) > 1:
```

Comment:
`msg` can be `None` or `len(msg)` can be 0. I think this method should rather look like this:

```python
def __init__(self, *args, **kwargs):
    if not args:
        self.msg = None
    elif len(args) > 1:
        self.msg = args[0] % args[1:]
    else:
        self.msg = args[0]
    self.logfile = kwargs.get('logfile')
```